scvi/model/base/_base_model.py

```diff
@@ -44,8 +41,8 @@ def __init__(self, adata: Optional[AnnData] = None):
         if adata is not None:
             self.adata_manager = self.get_anndata_manager(adata, required=True)
             self.adata = self.adata_manager.adata
-            self.scvi_setup_dict_ = self.adata_manager.get_setup_dict()
-            self.summary_stats = self.scvi_setup_dict_[_SUMMARY_STATS_KEY]
+            self.registry = self.adata_manager.registry
```

Comment:
should this be registry or data_registry? And if it's the latter should registry be a private attribute? I'm not sure I understand the distinction between the two in the manager